guest.speaking/MedicalDoctors4CovidEthicsIntl-CharlesKovess/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess]/v5hcyu4 (2024-10-22) - Dr J. Jay Couey discusses the human genome [Charles Kovess].diarized.vtt

```diff
@@ -2860,10 +2862,10 @@ WEBVTT
 [David Rasnick]: I'd be happy to do it.
 
 01:44:23.508 --> 01:44:26.249
-[David Rasnick]: Oh yes, can you remember what the topic is, though?
+[Stephen Frost]: Oh yes, can you remember what the topic is, though?
 
 01:44:26.309 --> 01:44:27.510
-[Stephen Frost]: Aneuploidy.
+[David Rasnick]: Aneuploidy.
```

Comment:
Wow... I don't even know what this word is and AI knows it. :sweat_smile:

Reply:
I know, impressive. I had to look it up. My vocabulary is always expanding since watching Jay's streams!